docs/releases.md

```diff
@@ -4,28 +4,66 @@
-- [**DietPi-Drive_Manager**](../dietpi_tools/#dietpi-drive-manager) :octicons-arrow-right-16: Transferring the root filesystem is now supported on Odroid C4/HC4 models.
-- [**DietPi-Drive_Manager**](../dietpi_tools/#dietpi-drive-manager) :octicons-arrow-right-16: The option to transfer the root filesystem is now shown as well when the drive does not contain any filesystem. The process implies formatting with a supported filesystem anyway which makes the condition obsolete. Also it is not needed anymore to unmount the filesystem manually first, as this is done automatically before formatting. In the related format menu, only supported filesystems are listed, i.e. ext4 and additionally F2FS on Raspberry Pi. The input box to enter a custom mount point is omitted as it is only temporarily used until the root filesystem transfer has finished. The swap file is not disabled anymore as it can be copied and reused as is on the new root filesystem without issues.
-- [**DietPi-Drive_Manager**](../dietpi_tools/#dietpi-drive-manager) :octicons-arrow-right-16: When transferring the root filesystem, further checks are performed: There needs to be a dedicated boot partition (required on Raspberry Pi and default on our current Odroid images), as this is what the transfer steps expect and the only reason for moving only the root filesystem instead of cloning the whole drive or flashing the DietPi image to an external drive in the first place. The expected boot/kernel configuration files need to be present so that the kernel can be instructed to mount the new root filesystem. The DietPi user data must not currently be located at the target partition as it would be formatted and all data hence lost.
-- [**DietPi-Drive_Manager**](../dietpi_tools/#dietpi-drive-manager) :octicons-arrow-right-16: On ext4 filesystems, the reserved blocks percentage is now shown and can be changed as well if the drive is not currently mounted.
-- [**DietPi-RAMlog**](../software/log_system/#dietpi-ramlog) :octicons-arrow-right-16: The /var/log directory structure is now synced to the persistent disk storage after software installs and via daily cron job to prevent missing logs files or directories in case of unclean shutdowns, which usually lead to failing service startups.
+- [**DietPi-Backup**](../dietpi_tools/#dietpi-backup-backuprestore)
+    - A feature has been added to enable daily system backup via cron job. Many thanks to @Lycidias93 for suggesting this feature: <https://github.com/MichaIng/DietPi/issues/3871>
+        ![DietPi Backup Daily](assets/images/dietpi-backup-daily.jpg){: width="713" height="300" loading="lazy"}
```

Comment:
No need to change, but for the future: Such console screens look much better as PNG and also the file size is smaller. JPEG seems to not deal well with material like sharp edges and non-fluent sharp colour cuts, while it's better for natural photos with many colours, organic colour gradients and contours.

Reply:
good idea. changed